doc/classes/CollisionObject2D.xml

```diff
@@ -206,6 +206,9 @@
 			The physics layers this CollisionObject2D scans. Collision objects can scan one or more of 32 different layers. See also [member collision_layer].
 			[b]Note:[/b] Object A can detect a contact with object B only if object B is in any of the layers that object A scans. See [url=$DOCS_URL/tutorials/physics/physics_introduction.html#collision-layers-and-masks]Collision layers and masks[/url] in the documentation for more information.
 		</member>
+		<member name="collision_weight" type="float" setter="set_collision_weight" getter="get_collision_weight" default="1.0">
```

Comment:
How about calling it "collision_priority" (or similar), the word weight can be confusing as a static body has infinite mass.